docs/getting-started-guides/windows/index.md

```diff
@@ -1,139 +1,144 @@
+Today, Windows CNI plugin is based on wincni.exe code with the following example, configuration file.
 
+Note: this file assumes that a user previous created 'l2bridge' host networks on each Windows node using <Verb>-HNSNetwork cmdlets as shown in the start-kublet.ps1 and start-kubeproxy.ps1 scripts linked above
 ```
-docker network create -d transparent --gateway 192.168.2.1 --subnet 192.168.2.0/24 <network name>
-# A bridge is created with Adapter name "vEthernet (HNSTransparent)". Set its IP address to transparent network gateway
-netsh interface ipv4 set address "vEthernet (HNSTransparent)" addr=192.168.2.1
-route add 192.168.0.0 mask 255.255.255.0 192.168.0.1 if <Interface Id of the Routable Ethernet Adapter> -p
-route add 192.168.1.0 mask 255.255.255.0 192.168.1.1 if <Interface Id of the Routable Ethernet Adapter> -p
+{
+    "cniVersion":  "0.2.0",
+    "name":  "l2bridge",
+    "type":  "wincni.exe",
+    "master":  "Ethernet",
+    "ipam":  {
+                 "environment":  "azure",
+                 "subnet":  "10.10.187.64/26",
+                 "routes":  [
+                                {
+                                    "GW":  "10.10.187.66"
+                                }
+                            ]
+             },
+    "dns":  {
+                "Nameservers":  [
+                                    "11.0.0.10"
+                                ]
+            },
+    "AdditionalArgs":  [
+                           {
+                               "Name":  "EndpointPolicy",
+                               "Value":  {
+                                             "Type":  "OutBoundNAT",
+                                             "ExceptionList":  [
+                                                                   "11.0.0.0/8",
+                                                                   "10.10.0.0/16",
+                                                                   "10.127.132.128/25"
+                                                               ]
+                                         }
+                           },
+                           {
+                               "Name":  "EndpointPolicy",
+                               "Value":  {
+                                             "Type":  "ROUTE",
+                                             "DestinationPrefix":  "11.0.0.0/8",
+                                             "NeedEncap":  true
+                                         }
+                           },
+                           {
+                               "Name":  "EndpointPolicy",
+                               "Value":  {
+                                             "Type":  "ROUTE",
+                                             "DestinationPrefix":  "10.127.132.213/32",
+                                             "NeedEncap":  true
+                                         }
+                           }
+                       ]
+}
```

Comment:
Different sections are definitely needed. This does not apply to all CNI plugins.

Reply:
Addressed in https://github.com/JMesser81/kubernetes.github.io/pull/1